charts/firefly/templates/_helpers.tpl

```diff
@@ -138,10 +138,12 @@ debug:
 {{- end }}
 http:
   port: {{ .Values.core.service.httpPort }}
+  publicURL: {{ .Values.core.service.httpPublicURL }}
```

Comment:
Like the idea of this, but needs to account for the fact that the Ingress could be the public URL if its enabled and the service URL is deterministic based on the release name.

Was thinking we'll want a helper similar to this one to facilitate that: https://github.com/hyperledger/firefly-helm-charts/blob/main/charts/firefly/templates/_helpers.tpl#L121-L130

Reply:
So to make this automatic we'd need a helper to:
- check if ingress is enabled
- check `host` on ingress definition
- check if `tls` has host listed on that to determine `http`/`https`
- allow a manual override if a config value is set

That about right?

Reply:
Ah bit more than I originally expected but yes that sounds great! Forgot w/ DX TLS is always enabled so there wasnt a determination for HTTP / S.

Not sure if we have to check if the length of the list for `host` is non-zero since Helm would throw an error if thats the case which to me is the desired user experience since they enabled an ingress without specifying a host.